src/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <PropertyGroup>
         <IsPackable>true</IsPackable>
         <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <LangVersion>latest</LangVersion>
```

Comment:
Isn't `latest` the default? What's the added benefit of this change?

Reply:
The default language version depends on the target framework, see the [C# language versioning](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-versioning#defaults) documentation.

For .NET Standard 2.0 the language version is C# 7.3 and for .NET Standard 2.1 the language version is C# 8.0.

I don't remember everything I changed but there's a high probability that I used some feature that requires C# greater than 7.3.

Reply:
My mistake.

But it may be different for different SDK versions.

But, if you're aiming for a version higher than the supported for that target, you need to be very careful.